lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -556,6 +557,17 @@ public void setAnimationFromUrl(String url, @Nullable String cacheKey) {
     setCompositionTask(task);
   }
 
+  /**
+   * Load a <a href="https://dotlottie.io/">dotLottie</a> animation from a ZipInputStream.
+   * <p>
+   * Auto-closes the stream.
+   */
+  public void setAnimationFromDotLottie(ZipInputStream stream, @Nullable String cacheKey) {
```

Comment:
```suggestion
  public void setAnimation(ZipInputStream stream, @Nullable String cacheKey) {
```

You can give this the same name. The method overloading specificity is enough here.

It's worth noting that dotLottie aren't the only zip files Lottie supports. You can zip any animations or animations + images so the naming/docs should not be dotLottie specific.

I was on a plane yesterday and actually drafted up a [similar PR](https://github.com/airbnb/lottie-android/pull/2411). I'm happy to land your contribution since you got it first, however.

Reply:
Makes sense! I don't have a strong opinion on which PR we land. Maybe yours, since I like your docs more 😄

Reply:
Okay, up to you. Feel free to update it or close it and then I'll land mine.